phc/__init__.py

```diff
@@ -1,3 +1,4 @@
 from phc.web.session import Session
 from phc.web.accounts import Accounts
 from phc.web.projects import Projects
+from phc.web.fhir_search import FhirSearch
```

Comment:
What if we just made this `Fhir` and could add CRUD methods to it later to go with the search?

Reply:
Sounds good.. I think thats the same pattern in the cli.